Makefile

```diff
@@ -205,7 +205,6 @@ release:
 	${MAKE} pack
 
 .PHONY=pack
-pack: YARNFLAGS=--check-cache
```

Comment:
It seems like --check-cache is not also not used on release builds. Is this intended? (I think it's not needed but want to make sure)

Reply:
Yes, that is intended. There is no reason to check the cache if it just got nuked and rebuilt anyways.

Reply:
Otherwise I think this could be merged.